modules/platforms/dotnet/Apache.Ignite.Core/Impl/Plugin/PluginProcessor.cs

```diff
@@ -145,18 +149,40 @@ public ExceptionFactory GetExceptionMapping(string className)
 
             ExceptionFactory res;
 
-            return _exceptionMappings.TryGetValue(className, out res) ? res : null;
+            if (_exactExceptionMappings.TryGetValue(className, out res))
+            {
+                return res;
+            }
+
+            foreach (var mapping in _patternExceptionMappings)
+            {
+                if (className.StartsWith(mapping.Key))
```

Comment:
`className.StartsWith(mapping.Key, StringComparison.Ordinal)`- use `Ordinal` comparison explicitly, otherwise `CurrentCulture` is used, which can have unexpected behavior with some cultures.

Reply:
Thanks, fixed.